### PR TITLE
Store changelogs in the repository

### PR DESCRIPTION
**Pull request by Totktonada**
See https://github.com/tarantool/tarantool/discussions/5634

### CODE REVIEW

File: tools/gen-release-notes [hunk 1]
```diff
@@ -0,0 +1,465 @@
...
+
+# }}} Templates
+
+
+# {{{ Collecting
+
+def changelog_entries_sorted(source_dir, entries_dir_relative):
+    """ Acquire order of appearance from ```git log``.
+
+        Misses uncommitted entries and contains files that was
```
Comment (avtikhon) [inline tools/gen-release-notes:141]: were

File: doc/changelogs.md [hunk 2]
```diff
@@ -0,0 +1,153 @@
...
+  Explicit `login:@host:port` was necessary before, otherwise a replica
+  displayed the following error:
+
+  > Missing mandatory field 'tuple' in request
+```
+
+## Leave a note to a release manager
+
+A valuable feature or a deliberate compatibility breakage may require an extra
+notice for a maintainer who will squash changelog entries into a a changelog
```
Comment (avtikhon) [inline doc/changelogs.md:55]: duplication of 'a'

File: tools/gen-release-notes (hunk 1 above)
Comment (Totktonada) [reply tools/gen-release-notes]: Thanks, fixed!

File: doc/changelogs.md (hunk 2 above)
Comment (Totktonada) [reply doc/changelogs.md]: Fixed.

**Review by avtikhon — APPROVED** [review]
Seems all correct, thanks for the patch, LGTM.

**Pull request merged**